tests/FsCheck.Test/Gen.fs

```diff
@@ -21,7 +21,20 @@ module Gen =
         lazy (  Gen.elements l
                 |> sample 50
                 |> List.forall (isIn l))
-    
+
+    [<Property>]
+    let GrowingElements (PositiveInt i) =
+        let sizes = [ 1..i ]
+        let run g =
+            [ for s in sizes ->
+                  g
+                  |> Gen.resize s
+                  |> Gen.eval   s (Random.newSeed()) ]
```

Comment:
Can you use `sequence` here turn a list of generators into a generator of a list, and then `sample` to generate the list?